src/components/listitems/ListItemTransaction.tsx

```diff
@@ -76,11 +72,13 @@ export type ListItemTransaction = WithTestID<
           transactionStatus: ListItemTransactionStatusWithoutBadge;
           badgeText?: string;
           transactionAmount: string;
+          transactionAmountAccessibilityLabel: string;
```

Comment:
Since we are going to introduce breaking changes, how about grouping the values that are related to the 'transaction' by defining a single object?
```
transaction: {
  amount: string;
  accessibilityLabel: string;
  status: ListItemTransactionStatusWithoutBadge;
}
```